src/framework/framework_application.js

```diff
@@ -591,6 +592,61 @@ pc.extend(pc, function () {
             }
         },
 
+        _fillFrameStats: function(now, dt, ms) {
+            // Timing stats
+            var stats = this.stats.frame;

```

Comment:
Can you name `stats` as `frame`, otherwise it is a bit miscommunication later on where it puts actual values.
Same for `drawCalls` later on.